.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: ruby
+rvm:
+ - 1.9.3
```

Comment:
Yes I think so, ruby 2x is supported by jruby 9.x - it's up to you if you think we need to support JRuby